build/functions.sh

```diff
@@ -206,7 +206,7 @@ function verify_and_set_go_version () {
     
     go_version=`$g version | awk '{print $3}'`
 
-    if [[ $go_version =~ go([1-9])\.([1-9]+) ]] && [[ ${BASH_REMATCH[1]} -ge 1 ]] && [[ ${BASH_REMATCH[2]} -ge 11 ]]; then
+    if [[ $go_version =~ go([1-9])\.([1-9]+) ]] && [[ ${BASH_REMATCH[1]} -ge 1 ]] && [[ ${BASH_REMATCH[2]} -ge 13 ]]; then
```

Comment:
this should be `-ge 14` now right?

Reply:
Oops, I missed that in the rebase. Fixed in ea5e3e57c3.